docs/source/user_guide/installation/index.rst

```diff
@@ -50,8 +50,165 @@ Following GNU/Linux and macOS solvers are optionally available:
+Dependency                                                  Minimum Version    pip extra          Notes
+=========================================================== ================== ================== ===================================================================================================================================================================================
+`sympy <https://docs.sympy.org/latest/index.html>`__        1.8.0              latexify           For symbolic mathematics.
+=========================================================== ================== ================== ===================================================================================================================================================================================
+
+.. _install.bpx_dependencies:
+
+bpx dependencies
+^^^^^^^^^^^^^^^^^
+
+Installable with ``pip install "pybamm[bpx]"``
+
+=========================================================== ================== ================== ===================================================================================================================================================================================
+Dependency                                                  Minimum Version    pip extra          Notes
+=========================================================== ================== ================== ===================================================================================================================================================================================
+`bpx <https://pypi.org/project/bpx/>`__                     -                  bpx                Battery Parameter eXchange
+=========================================================== ================== ================== ===================================================================================================================================================================================
+
+.. _install.tqdm_dependencies:
+
+tqdm dependencies
+^^^^^^^^^^^^^^^^^
+
+Installable with ``pip install "pybamm[tqdm]"``
+
+=========================================================== ================== ================== ===================================================================================================================================================================================
+Dependency                                                  Minimum Version    pip extra          Notes
+=========================================================== ================== ================== ===================================================================================================================================================================================
+`tqdm <https://tqdm.github.io/>`__                          -                  tqdm               For faster loops.
```

Comment:
does it really speed up loops?? I think it's just for logging loops

Reply:
I got it wrong, thanks for correction 😃